covasim/webapp/cova_app.css

```diff
@@ -121,6 +121,7 @@ fieldset > legend {
     display: block;
     position: relative !important;
     width: initial;
+    /*overflow-y: scroll; CK: for later use */
```

Comment:
Still would really, really like this but not required for RC1, and adding this breaks the layout